test/test.jl

```diff
@@ -4,10 +4,10 @@ function profile_test(n)
     for i = 1:n
         A = randn(100,100,20)
         m = maximum(A)
-        Afft = fft(A)
-        Am = mapslices(sum, A, 2)
+        # Afft = fft(A)
```

Comment:
I assume the `fft` call is only intended to provide some nontrivial amount of computations that can be profiled. With FFTW having been moved out of Base, it might be better to replace it with something that doesn't require a testing dependency. Or is it fine to just skip it?

Reply:
Agreed better not to use `fft`. You can either replace it or skip it, either is fine with me.